python/cugraph/cugraph/community/egonet.py

```diff
@@ -86,21 +91,54 @@ def ego_graph(G, n, radius=1, center=True, undirected=False, distance=None):
     >>> ego_graph = cugraph.ego_graph(G, 1, radius=2)
 
     """
-
     (G, input_type) = ensure_cugraph_obj(G, nx_weight_attr="weight")
-    result_graph = type(G)()
 
-    if G.renumbered is True:
-        if isinstance(n, cudf.DataFrame):
-            n = G.lookup_internal_vertex_id(n, n.columns)
-        else:
-            n = G.lookup_internal_vertex_id(cudf.Series([n]))
+    result_graph = type(G)()
```

Comment:
Does the result graph need to be directed/undirected the same as the input graph?  If so, you'll need to do this:
```suggestion
    result_graph = type(G)(directed=G.is_directed())
```

Reply:
The C++ code will make the graphs match.  That is, if the input graph is undirected then the resulting graph is undirected.  If the input graph is directed then the resulting graph will be directed.

Making them match (as Rick suggested) seems like the right thing to do.

Reply:
I concur . Thanks